### PR TITLE
Update tutorials.rst

### PR DESCRIPTION
**Pull request by dani-swordfish**
I removed two dead links and added two resources I found helpful when learning.

A possible improvement could be to label 4.0 tutorials and remove or archive pages which have had no new content in 3-4 years.

### CODE REVIEW

File: community/tutorials.rst [hunk 1]
```diff
@@ -38,21 +38,21 @@ Video tutorials
...
 - `Code with Tom <https://www.youtube.com/playlist?list=PLiUQR4U_J9ec0k91iHPme_qtfS1nrWF3W>`_ (2D and 3D, GDScript).
 - `BornCG <https://www.youtube.com/playlist?list=PLda3VoSoc_TSBBOBYwcmlamF1UrjVtccZ>`_ (2D and 3D, GDScript).
-- `Gonkee <https://www.youtube.com/channel/UCJqCPFHdbc6443G3Sz6VYDw>`_ (2D and 3D, GDScript, Shaders).
 - `TheGuideKnight <https://www.youtube.com/playlist?list=PLYcTJBw32KtX2fvOdtO_fS3Nd6sxM71tn>`_ (2D, GDScript).
 - `GDScript Dude <https://www.youtube.com/channel/UCQs0i6vKISElM6mh7OzLouQ>`_ (GDScript).
 - `Garbaj <https://www.youtube.com/c/Garbaj/>`_ (3D, GDScript).
 - `Kasper Frandsen <https://www.youtube.com/c/KasperFrandsen/>`_ (3D, Shaders).
 - `bitbrain <https://www.youtube.com/watch?v=lFIBn8kJ-IM&list=PL4AhUX6lGjJWfiDORSmEu8x_myelbe1p2>`_ (2D, GDScript).
 - `Gwizz <https://www.youtube.com/@Gwizz1027>`_ (2D, GDScript).
+- `Quiver <https://quiver.dev/>`_ (2D, GDScript, Game Jams).
```
Comment (skyace65) [inline community/tutorials.rst:47]:
```suggestion
- `Quiver <https://quiver.dev/>`_ (2D, GDScript).
```
I'm not sure if "Game Jams" is something we want to have labeled for this page. I'd like a second opinion on this, I'm not against it but I don't see it as particularly helpful.

**Review by skyace65 — COMMENTED** [review]
Confirmed the two dead links are Dead. Didn't get to do an in depth look at the two new ones but they're probably fine. Need a second opinion on if we want to label game jams

File: community/tutorials.rst (hunk 1 above)
Comment (dani-swordfish) [reply community/tutorials.rst:47]: Your change makes sense to me. Thanks.

**Comment by skyace65** [comment]
Thanks! And congrats on your first merged PR!

**Pull request merged**